frontend/test/__support__/store.ts

```diff
@@ -56,7 +48,42 @@ export interface EntitiesStateOpts {
   questions?: Card[];
 }
 
+const EntitiesSchema: Record<keyof EntitiesState, NormalizrSchema<any>> = {
+  actions: [ActionSchema],
+  alerts: [],
```

Comment:
These types should make TypeScript throw an error if we don't provide an explicit schema for a new entity added to `EntitiesState`. Not every entity needs a normalizr schema, we can just use `[]` in that case as we do with `alerts`